src/main/java/qna/domain/Question.java

```diff
@@ -108,7 +111,7 @@ public String toString() {
                 "id=" + id +
                 ", title='" + title + '\'' +
                 ", contents='" + contents + '\'' +
-                ", writerId=" + writerId +
+                ", user=" + writer +
```

Comment:
```suggestion
                ", writer=" + writer +
```

연관 관계에서 toString으로 출력 시 고려하신 부분이 있을까요?